src/ape/managers/networks.py

```diff
@@ -133,13 +206,37 @@ def parse_network_choice(
         network_choice: Optional[str] = None,
         provider_settings: Optional[Dict] = None,
     ) -> ProviderContextManager:
+        """
+        Parse a network choice into a context manager for managing a temporary
+        connection to a provider. See
+        :meth:`~ape.managers.networks.NetworkManager.network_choices` for all
+        available choices (or use CLI command ``ape networks list``).
+
+        Args:
+            network_choice (str, optional): The network choice
+              (see :meth:`~ape.managers.networks.NetworkManager.network_choices`).
+              Defaults to the default ecosystem, network, and provider combination.
+            provider_settings (dict, optional): Settings for the provider. Defaults to None.
+
+        Returns:
+            :class:`~api.api.networks.ProviderContextManager`
+        """
+
         provider = self.get_provider_from_choice(
             network_choice, provider_settings=provider_settings
         )
         return ProviderContextManager(self, provider)
 
     @property
     def default_ecosystem(self) -> EcosystemAPI:
+        """
+        The default ecosystem. Call
+        :meth:`~ape.managers.networks.NetworkManager.set_default_ecosystem` to
+        change the default ecosystem. If a default is not set and there is
+        only a single ecosystem installed, such as Ethereum, then returns
+        that ecosystem.
```

Comment:
Note: probably want a way to actually specify a default in `ape-config.yaml`

Reply:
I just created an issue: https://github.com/ApeWorX/ape/issues/321